idea_town/frontend/templates/idea_town/frontend/index.html

```diff
@@ -3,6 +3,9 @@
 <head>
     <title>Idea Town</title>
     <link rel="stylesheet" href="{{ static('styles/main.css') }}">
+    <link rel="x-current-user" href="{{ url('users_self') }}">
+    {# TODO: Make this a dev / prod setting? Or, expose in an API endpoint? #}
```

Comment:
Once nice thing about using an API endpoint: the addon can reuse it to look up experiment addon URLs. But this is fine for now, if you just want to land it as-is